Makefile

```diff
@@ -13,6 +13,7 @@ STAGE1 := shecc-stage1.elf
 STAGE2 := shecc-stage2.elf
 
 OUT ?= out
+ARCH := arm
```

Comment:
Should be `ARCH ?= arm`

Reply:
It has not been set and used before. I think it should be fine?

Reply:
No, the variable may be set in automated build environments such as CI pipeline.